backend/api/curious/curiousApi.test.ts

```diff
@@ -1,18 +1,198 @@
-import CuriousApi, { dummyLearnerProfiles, dummyLearnerLatestAssessments } from './curiousApi'
+import nock from 'nock'
```

Comment:
Just noticed you have put the tests in the same dir as the source. We generally don't do that, so i would prefer to keep it like that unless there is a good reason (you probably have one).